Firestore/core/src/firebase/firestore/core/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ cc_library(
   SOURCES
     database_info.cc
     database_info.h
+    snapshot_version.cc
```

Comment:
SnapshotVersion is in the model package on Android and we should have fixed this on iOS but didn't. Can you please move this `model` to line up with Android?

Reply:
done